src/components/common/zoom_button/zoom_button.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+
```

Comment:
@Sal-Meza I'd clean-up some of the extra lines here. Typically, you should only do a single blank line when breaking up sections of the code. I'd condense the top to a single blank line, and then would remove the blank line between the comments and the const export.